tests/utilities/test_model_helpers.py

```diff
@@ -92,3 +94,24 @@ def on_fit_start(self):
     trainer = Trainer(fast_dev_run=1)
     trainer.fit(model, train_dataloader=model.train_dataloader())
     assert model.on_fit_start_called
+
+
+def test_get_model_size():
+    model = BoringModel()
+
+    size_bytes = get_model_size(model)
+    assert size_bytes == 0.001319
```

Comment:
As we want to keep the original implementation of model_size, changed @calebrob6's implementation from using bytes to megabytes.